src/util/isGif.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(url){
+  if(url.match){
+    const toMatch = url.match('data:image/(.*);') || 'png';
```

Comment:
This is a regex right? If that is the case, then can you use regex literals `//`? Put the regex between `/` and `/`.

Reply:
Yes it is working fine, but I'll add a `\` before `/`.

Reply:
I'm sorry could you explain a bit :sweat_smile: I don't understand!

Reply:
JavaScrip has dedicated regex literals just like string literals and template literals
```js
const string = 'This is a string literal. It is just a string'
const templateString = `This is a template literal.
It is a string with superPowers ${variable}`

const regex = /^g$/ig; // This is a regex literal. It is used to store regex (along with options like global (`g`) and case insensitive (`i`).

Reply:
Ohh! I did not knew that! Thanks! :astonished:

Reply:
Done!